modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js

```diff
@@ -20,9 +20,39 @@ import * as admin from '../../page-models/pageAdminListOfRegisteredUsers';
 import {pageProfile as profile} from '../../page-models/pageProfile';
 import * as notifications from '../../components/permanentNotifications';
 import {userMenu} from '../../components/userMenu';
+import { Selector } from 'testcafe';
+import { successNotification } from '../../components/notifications';
+import {confirmation} from '../../components/confirmation'
 
 const regularUser = createRegularUser();
 
+const _checkListOfActions = async(t, actions, excluded) => {
+    const actionSelector = Selector('.dropdown-menu a');
+
+    for (let i = 0; i < actions.length; i++) {
+        const actionLbl = actions[i];
+
+        if (excluded)
+            await t.expect(actionSelector.withText(actionLbl).exists).notOk();
+        else
+            await t.expect(actionSelector.withText(actionLbl).exists).ok();
```

Comment:
```js
await t.expect(actionSelector.withText(actionLbl).exists == excluded).notOk();
```

Reply:
TestCafe-specific promises don't work like that.

Reply:
Changed check of action list.